pkg/target/target.go

```diff
@@ -260,11 +260,11 @@ func (h *K8sValidationTarget) GetCache() handler.Cache {
 // setObjectOnDelete enforces that we use at least K8s API v1.15.0+ on DELETE operations
 // and copies over the oldObject into the Object field for the given AdmissionRequest.
 func setObjectOnDelete(review *gkReview) error {
-	if review.AdmissionRequest.Operation == admissionv1.Delete {
+	if review.Operation == admissionv1.Delete {
 		// oldObject is the existing object.
```

Comment:
gkReview != AdmissionRequest
review does not have the same fields as admissionRequest
https://github.com/open-policy-agent/gatekeeper/blob/b371688b39055fc1d32e1a0014f3627e4d11cc16/pkg/target/target.go#L103-L107

Please revert these changes

Reply:
`AdmissionRequest` is embedded struct within `gkReview` so its fields are available under `gkReview`

Reply:
`AdmissionRequest` is embedded in `gkReview` so field of `AdmissionRequest` should be accessible direcly.